src/sqlfluff/dialects/dialect_clickhouse.py

```diff
@@ -564,13 +664,78 @@ class CreateMaterializedViewStatementSegment(BaseSegment):
     )
 
 
+class CreateDatabaseStatementSegment(ansi.CreateDatabaseStatementSegment):
+    """A `CREATE DATABASE` statement.
+
+    As specified in
+    https://clickhouse.com/docs/en/sql-reference/statements/create/database
+    """
+
+    type = "create_database_statement"
+
+    match_grammar = Sequence(
+        "CREATE",
+        "DATABASE",
+        Ref("IfNotExistsGrammar", optional=True),
+        Ref("DatabaseReferenceSegment"),
+        AnySetOf(
+            Sequence(
+                "ON",
+                "CLUSTER",
+                Ref("ExpressionSegment"),
+                optional=True,
+            ),
+            Ref("DatabaseEngineSegment", optional=True),
+            Sequence(
+                "COMMENT",
+                Ref("ExpressionSegment"),
```

Comment:
This doesn't need to be an expression

Reply:
How may I detect a `COMMENT` pattern? (also for `ON CLUSTER`)

Comment might be commented string and single unquoted word.